packages/webcert/src/utils/TextWithDynamicLinks.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { configureStore, EnhancedStore } from '@reduxjs/toolkit'
+import { render, screen } from '@testing-library/react'
+import faker from 'faker'
+import { createMemoryHistory } from 'history'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router-dom'
+import { certificateMiddleware } from '../store/certificate/certificateMiddleware'
+import reducer from '../store/reducers'
+import dispatchHelperMiddleware from '../store/test/dispatchHelperMiddleware'
+import TextWithDynamicLinks from './TextWithDynamicLinks'
+let testStore: EnhancedStore
+
+const history = createMemoryHistory()
+
+const text1 = faker.lorem.sentence(10)
+const text2 = faker.lorem.sentence(10)
+const text3 = faker.lorem.sentence(10)
+const link1 = faker.lorem.word()
+const link2 = faker.lorem.word()
+
+const textWithLinks = `${text1} <LINK:${link1}> ${text2} <LINK:${link2}> ${text3}`
+
+const renderComponent = () => {
+  render(
+    <Provider store={testStore}>
```

Comment:
testStore, Provider etc. is not really necessary for this component since it is not relying on redux functionality.

Reply:
This is not resolved. Provider, testStore and middleware etc. can all be removed from tests for this component :)

Reply:
These are needed, since WCDynamicLink  relies on redux. Tests fails without them

Reply:
Oh, you are correct. Perhaps add a test that populates for example link1 with a dynamic key using `updateDynamicLinks` and test that it shows up.

Reply:
Added test than link texts are rendered